### PR TITLE
docs: add correct date format to clipboards of Milestones-json

### LINKED ISSUE #9183: [DOCS] incorrect date format in "Milestones with JSON" code snippet
### Description

I noticed the JSON snippet for the documentation on "Milestones with JSON" is somewhat ambiguous regarding the date format. It took me a while to realize that the correct date format is `Day/Month/Year` rather than `Month and Year`.

### Screenshots

image

Found the correct format on Eddie's biodrop profile:
image

### Do you want to work on this issue?

Yes

### Additional information

I'm not well-versed in Node.js, but I'd like to attempt to resolve this issue. I'd also appreciate it if I could be paired with someone proficient in Nodejs to help solve it. Thanks

### PR DESCRIPTION
**Pull request by bL34cHig0**
Closes  #9183

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by accodes21 — COMMENTED** [review]
looks good

**Comment by bL34cHig0** [comment]
Thanks @accodes21

File: pages/docs/how-to-guides/milestones-json.mdx [hunk 1]
```diff
@@ -31,7 +31,7 @@ _If you need help on how to edit this file, please see the <Link href="/docs/how
 | Property    | Required | Description                                                 |
 | :---------- | :------- | :---------------------------------------------------------- |
 | title       | true     | Main heading of the milestone                               |
-| date        | true     | Month and Year                                              |
+| date        | true     | Month/Day/Year                                              |
```
Comment (eddiejaoude) [inline pages/docs/how-to-guides/milestones-json.mdx:34]: This should be 

```suggestion
| date        | true     | Day/Month/Year                                              |
```

**Comment by bL34cHig0** [comment]
Thank you @eddiejaoude for correcting me on this.

**Pull request merged**